src/assets/githubAction/githubActionBeginner.md

```diff
@@ -0,0 +1,130 @@
+---
+title: GitHub Actions - CI & CD
+authorName: Nabin Ale
+authorAvatar : https://avatars.githubusercontent.com/u/61624650?s=400&u=8f992a9ca189af19c40cb070a5af1e02669d3884&v=4
+authorLink: https://github.com/nabim777
+createdAt: Oct 30, 2023
+tags: CI, CD, github, githubAction
+banner: https://blog.jankaritech.com/src/assets/githubAction/images/githubaction_banner.png
+---
+GitHub Actions is an automation and CI/CD (Continuous Integration/Continuous Delivery or Deployment) service provided by GitHub, allowing us to automate our software development processes or workflows, from building and testing code to deploying it to different environments. It's an effective tool that can enhance software quality and save our time.
+
+## Some features of GitHub Actions:
+- It allows you to automate your Software Development Life Cycle (SDLC) processes or workflows.
+- It simplifies CI/CD integration, facilitating seamless DevOps practices.
+
+## Components of GitHub Actions:
+1. Workflows
+2. Events
+3. Jobs
+4. Steps
+5. Actions
+6. Runners
+
+![components](/src/assets/githubAction/images/components_of_github_action.png)
+
+
+### 1. Workflows:
```

Comment:
```suggestion
### 1. Workflows
```
Sorry, I didn't notice this before. In English you usually use the double point if you have a list following (often not even formatted as a list) or if it is kind of like a subtitle, but not formatted as a title (like `Note:` on line 45)